packages/button/src/button.css

```diff
@@ -25,3 +25,24 @@ governing permissions and limitations under the License.
 ::slotted([slot='icon']) {
     inset: unset;
 }
+
+#container {
+    position: relative;
+    display: flex;
+}
+
+sp-progress-circle {
+    display: none;
+}
+
+:host([pending][disabled]) sp-progress-circle {
+    display: block;
+    position: absolute;
+    left: 50%;
+    transform: translate(-50%, 0);
+}
+
+:host([pending][disabled]) slot[name='icon'],
+:host([pending][disabled]) #label {
+    visibility: hidden;
```

Comment:
Can't remember what visibly hiding this content does to the accessibility tree right off, so making a note to confirm.

Reply:
Checked in voiceover and the label seems to be hidden? Is there some other way I can test?